rfc8446.xml

```diff
@@ -2993,9 +2996,10 @@ As noted above, the HMAC input can generally be implemented by a running
 hash, i.e., just the handshake hash at this point.</t>
 
 <t>In previous versions of TLS, the verify_data was always 12 octets long. In
-TLS 1.3, it is the size of the HMAC output for the Hash used for the handshake.</t>
+TLS 1.3, it is the size of the HMAC output for the Hash used for the handshake.
+Implementations receiving a verify_data with length shorter than 12 octets MUST abort the handshake with an "insufficient_security" alert.</t>
```

Comment:
The general thought here is "we're now negotiating a thing -- remember how TLS 1.2 lets you negotiate pure md5?"
I *think* that we have later text about HKDF output used in cases that need collision resistance that implicitly covers this scenario, so I don't expect we'll need to take this chunk.